documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,87 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
```

Comment:
@jcvenegas Afaik they use "systemd" to refer to the project themselves. https://www.freedesktop.org/wiki/Software/systemd/